<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>360</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Porypal</string>
  </property>
  <layout class="QVBoxLayout" name="main_layout">
   <property name="spacing">
    <number>15</number>
   </property>
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="topMargin">
    <number>15</number>
   </property>
   <property name="rightMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>15</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="control_layout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QGroupBox" name="button_group">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="title">
        <string>Image Processing</string>
       </property>
       <layout class="QHBoxLayout" name="processing_layout">
        <item>
         <widget class="QPushButton" name="btn_load_tileset">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>136</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>üß© Tileset Editor</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_load_image">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>115</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>üñºÔ∏è Load Image</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_save_image">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>94</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>üíæ Save Converted Image</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_extract_palette">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>94</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>üé® Extract palette from image</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_toggle_theme">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>üí° Toggle Theme</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="content_layout">
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="original_image_layout">
       <item>
        <widget class="QLabel" name="label_original">
         <property name="text">
          <string>Input Sprite:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGraphicsView" name="original_view">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>288</height>
          </size>
         </property>
         <property name="renderHints">
          <set>QPainter::Antialiasing</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="dynamic_layout">
       <property name="topMargin">
        <number>19</number>
       </property>
       <item>
        <widget class="QLabel" name="label_instruction">
         <property name="text">
          <string>Green = Selected Image | Blue = Recommended Image(s)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGraphicsView" name="dynamic_view"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
